kale/predict/class_domain_nets.py

```diff
@@ -126,30 +128,48 @@ def forward(self, input):
 # For Video/Action Recognition, DataClassifier.
 class ClassNetVideo(nn.Module):
     """Regular classifier network for video input.
-
     Args:
+        dict_n_class (dict, optional): the dictionary of class number for specific dataset.
         input_size (int, optional): the dimension of the final feature vector. Defaults to 512.
-        n_channel (int, optional): the number of channel for Linear and BN layers.
+        n_verb_channel (int, optional): the number of channel for Linear and BN layers for verb class.
+        n_noun_channel (int, optional): the number of channel for Linear and BN layers for noun class.
         dropout_keep_prob (int, optional): the dropout probability for keeping the parameters.
-        n_class (int, optional): the number of classes. Defaults to 8.
+        class_type (string): the type of class. Option=["verb", "verb+noun"]
     """
 
-    def __init__(self, input_size=512, n_channel=100, dropout_keep_prob=0.5, n_class=8):
+    def __init__(
+        self,
+        dict_n_class,
+        input_size=512,
+        n_verb_channel=256,
+        n_noun_channel=512,
+        dropout_keep_prob=0.5,
+        class_type="verb",
```

Comment:
Similar to my comments above, we can implement the a class for the type of classes.